contrib/runners/actionchainrunner/runner.yaml

```diff
@@ -0,0 +1,14 @@
+- aliases: []
+  description: A runner for launching linear action chains.
+  enabled: true
+  name: action-chain
+  runner_module: st2actions.runners.actionchainrunner
+  runner_parameters:
```

Comment:
We should call it parameters and in the code we can call it runner_parameters. UX feedback.